### PR TITLE
Update to Bot API 6.7

### PR DESCRIPTION
**Pull request by KnorpelSenf**
https://core.telegram.org/bots/api#april-21-2023

### CODE REVIEW

File: site/docs/README.md [hunk 1]
```diff
@@ -94,8 +94,8 @@ Works! :tada:
 
 <div style="font-size: 0.75rem; display: flex; justify-content: center;">
 
-© 2021-2023 &middot; grammY supports Telegram Bot API 6.6 which was [released](https://core.telegram.org/bots/api#march-9-2023) on March 9, 2023.
-(Last highlight: Translated Bot Descriptions)
+© 2021-2023 &middot; grammY supports Telegram Bot API 6.7 which was [released](https://core.telegram.org/bots/api#april-XX-2023) on April XX, 2023.
+(Last highlight: multiple bot names, custom emoji, and better inline queries)
```
Comment (KnorpelSenf) [inline site/docs/README.md:98]: If you can think of a better way to summarise this messy changelog, please commit

**Review by KnightNiwrem — APPROVED** [review]

File: site/docs/plugins/keyboard.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ For example, it is not possible to first send a custom keyboard along with a mes
 grammY has a simple and intuitive way to build up the inline keyboards that your bot can send along with a message.
 It provides a class called `InlineKeyboard` for this.
 
-> Both `switchInline` and `switchInlineCurrent` buttons start inline queries.
+> The buttons added by calling `switchInline`, `switchInlineCurrent`, and `switchInlineChosenChat` start inline queries.
```
Comment (roj1512) [inline site/docs/plugins/keyboard.md:34]: What's this?

Comment (KnorpelSenf) [reply site/docs/plugins/keyboard.md:34]: An outdated name

**Comment by KnorpelSenf** [comment]
Are you aware that this was not released yet?

**Comment by KnorpelSenf** [comment]
Now it is actually

**Comment by KnorpelSenf** [comment]
nvm

**Comment by roj1512** [comment]
No, I wasn’t aware.

**Comment by KnorpelSenf** [comment]
We need to build the site again, several updates were released now

**Comment by roj1512** [comment]
Done!

**Comment by KnorpelSenf** [comment]
Yet https://grammy.dev/plugins/menu.html#introduction is still outdated

**Comment by roj1512** [comment]
I don’t think so, is it?

**Comment by KnorpelSenf** [comment]
Now it isn't, but it was at the time I wrote the comment. Glad it's fixed

**Comment by roj1512** [comment]
I dispatched the workflow and commented directly, not your fault. Takes some time :)

**Pull request merged**